SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -40,7 +40,8 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject
 	void SendComponentInterest(Worker_EntityId EntityId, const TArray<Worker_InterestOverride>& ComponentInterest);
 	FString GetWorkerId() const;
 	const TArray<FString>& GetWorkerAttributes() const;
-	Worker_RequestId SendEntityQueryRequest(const Worker_EntityQuery* EntityQuery);
+	Worker_RequestId SendEntityQueryRequest(const Worker_EntityQuery* EntiyQuery);
```

Comment:
Could you move these two function declarations to be under `SendComponentInterest`?

Reply:
The SendEntityQueryRequest function doesn't have anything to do with this PR. I think the reason the line was highlighted was due to a merge mistake I must have made a while back, which I have since fixed. I've moved SendMetrics (also moved it in the cpp file, I'm assuming you meant that too). Do you still want me to move the EntityQueryRequest function as well?

Reply:
Yes I do. My comment more of an organisational change that we can do while this file is changing.

Reply:
Got it, will do.

Reply:
Do you want the cpp file to change in order as well? I don't know how coding convention works for the order of functions in header files matching cpp files?

Reply:
Sure

Reply:
Alright it should all be done.